src/main/java/org/jitsi/videobridge/transform/RtxTransformer.java

```diff
@@ -345,4 +416,40 @@ private boolean encapsulateInRtxAndTransmit(
 
         return true;
     }
+
+    /**
+     * Returns the SSRC paired with <tt>ssrc</tt> in an FID source-group, if
+     * any. If none is found, returns -1.
+     *
+     * @return the SSRC paired with <tt>ssrc</tt> in an FID source-group, if
+     * any. If none is found, returns -1.
+     */
+    private long getFidPairedSsrc(RawPacket pkt)
```

Comment:
And this can be something like getPrimarySsrc (although I can't come up with something that makes the distinction between searching the current channel and searching all other channels)